AmplifyPlugins/Core/AWSPluginsCore/Model/Support/QueryPredicate+GraphQL.swift

```diff
@@ -60,12 +99,38 @@ extension QueryPredicate {
 }
 
 extension QueryPredicateOperation: GraphQLFilterConvertible {
+    func graphQLFilter(_ modelSchema: ModelSchema) -> GraphQLFilter {
+        let filterValue = [self.operator.graphQLOperator: self.operator.value]
+        return [modelSchema.columnName(forField: field): filterValue]
+    }
+
     var graphQLFilter: GraphQLFilter {
         return [field: [self.operator.graphQLOperator: self.operator.value]]
     }
 }
 
 extension QueryPredicateGroup: GraphQLFilterConvertible {
+    func graphQLFilter(_ modelSchema: ModelSchema) -> GraphQLFilter {
```

Comment:
the public method should not take an optional ModelSchema, but you can replace the computed property with the internal method taking in an operational model schema, right? this would reduce the duplication of code